docs/management/index-patterns.asciidoc

```diff
@@ -23,79 +18,93 @@ image::images/management-index-read-only-badge.png[Example of Index Pattern Mana
+filter your data by time. If you don’t specify a field, you won’t be able 
+to use the time filter.
 
-. Go to the *Settings > Indices* tab.
-. Select the pattern you want to set as the default in the Index Patterns list.
-. Click the pattern's *Favorite* button.
+Once you’ve created your index pattern, you can start working with 
+your {es} data in {kib}.  Here are some things to try:
 
-NOTE: You can also manually set the default index pattern in *Advanced > Settings*.
+* Interactively explore your data in Discover.
+* Present your data in charts, tables, gauges, tag clouds, and more in Visualize.
+* Show off your data in a Canvas presentation.
+* If your data includes geo data, visualize it using Maps.
+
+For a walkthrough of creating an index pattern and visualizing the data, 
+see <<getting-started, Getting Started>>. 
 
 [float]
-[[reload-fields]]
-== Reloading the Index Fields List
-When you add an index mapping, Kibana automatically scans the indices that match the pattern to display a list of the
-index fields. You can reload the index fields list to pick up any newly-added fields.
+==== Rolled up index pattern
 
-Reloading the index fields list also resets Kibana's popularity counters for the fields. The popularity counters keep
-track of the fields you've used most often within Kibana and are used to sort fields within lists.
+Clicking *Create index pattern* includes an item for creating a rollup 
+index pattern, if a rollup index is detected in the cluster. You create an 
+index pattern for rolled up data the same way you do for any data.  
 
-To reload the index fields list:
+You can match an index pattern to only rolled up data, or mix both rolled 
+up and raw data to visualize all data together. An index pattern can match 
+only one rolled up index, not multiple. There is no restriction on the 
+number of standard indices that an index pattern can match.
+With a rollup index, the time filter field is the same field used for 
+the rolled up date histogram aggregation.
 
-. Go to the *Settings > Indices* tab.
-. Select an index pattern from the Index Patterns list.
-. Click the pattern's *Reload* button.
+See <<visualize-rollup-data, Creating a visualization using rolled up data>> 
+for more detailed information.
 
 [float]
-[[delete-pattern]]
-== Deleting an Index Pattern
-Deleting an index pattern removes the pattern from the list of Saved Objects in Kibana.
-You will not be able to recover field formatters, scripted fields, source filters, and field popularity data
-associated with the index pattern.
+=== Managing your index patterns
+
+Once you’ve created an index pattern, you’re presented a list of all fields 
+and associated data type in the index.  
+
+[role="screenshot"]
+image:management/index-patterns/images/new-index-pattern.png["Index files and data types"]
+
+From this page, you can perform actions on the index pattern using the 
+controls in the upper right. You can:
```

Comment:
Would it make sense to remove `You can:` and move the colon after `right`. For example:

`From this page, you can perform actions on the index pattern using the controls in the upper right:`

The double `you can`s tripped me up.